app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -49,6 +58,15 @@ public User registerUser(UserRegistrationData registrationData) {
         return user;
     }
 
+    /**
+     * 유저의 정보를 수정하고 반환합니다.
+     *
+     * @param id 유저 식별자
+     * @param modificationData 수정할 유저의 정보
+     * @param userId 요청을 보낸 유저의 식별자
+     * @return 수정된 유저
+     * @throws AccessDeniedException 요청을 보낸 유저와 수정될 유저가 다르면 접근을 거부합니다.
```

Comment:
`@throws`는 케이스에 대한 소개 정도로 표현하는 것이 적절합니다.

```suggestion
     * @throws AccessDeniedException 요청을 보낸 유저와 수정될 유저가 다른 경우
```